cmd/osm-bootstrap/osm-bootstrap.go

```diff
@@ -175,6 +172,50 @@ func main() {
 		namespace:    osmNamespace,
 	}
 
+	crd_files, err := filepath.Glob("/osm-crds/*.yaml")
+	
+    if err != nil {
+        log.Fatal().Err(err).Msgf("error reading files from /osm-crds")
+    }
+
+	scheme = runtime.NewScheme()
+	codecs := serializer.NewCodecFactory(scheme)
+	decode := codecs.UniversalDeserializer().Decode
+
+    for _, file := range crd_files {
+		yaml, err := os.ReadFile(filepath.Clean(file))
+		if err != nil {
+			log.Fatal().Err(err).Msgf("Error reading CRD file %s", file)
+		}
+
+		crd := &apiv1.CustomResourceDefinition{}
+		_, _, err = decode(yaml, nil, crd)
+		if err != nil {
+			log.Fatal().Err(err).Msgf("Error decoding CRD file %s", file)
+		}
+
+		crd.Labels[constants.ReconcileLabel] = fmt.Sprintf("%t", enableReconciler)
+
+		crdExisting, err := crdClient.CustomResourceDefinitions().Get(context.Background(), crd.Name, metav1.GetOptions{})
+		if apierrors.IsNotFound(err) {
+			if _, err = crdClient.CustomResourceDefinitions().Create(context.Background(), crd, metav1.CreateOptions{}); err != nil {
+				log.Fatal().Err(err).Msgf("Error creating crd : %s", crd.Name)
+			}
+			log.Info().Msgf("Successfully created crd: %s", crd.Name)
+		} else {
+			log.Info().Msgf("Patching conversion webhook configuration for crd: %s, setting to \"None\"", crd.Name)
+
+			crdExisting.Spec = crd.Spec
```

Comment:
nit: let's set the reconciler label here for posterity, and in case it has changed